packages/framework/attributor/src/gzipEncoder.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { gzip, ungzip } from "pako";
+import { bufferToString, stringToBuffer } from "@fluidframework/common-utils";
+import { Jsonable } from "@fluidframework/datastore-definitions";
+import { Encoder } from "./encoders";
+
+export function makeGzipEncoder<T>(): Encoder<Jsonable<T>, string> {
```

Comment:
future PR should probably switch this to use lz4 since we already depend on lz4js. I think the diff is more readable without having to adjust package.json though, so for now would prefer to leave it this way.